GraknClient.java

```diff
@@ -361,16 +381,19 @@ public Keyspace keyspace() {
         }
 
         public List<AnswerGroup<ConceptMap>> execute(GraqlGet.Group query, boolean infer) {
-            return stream(query, infer).collect(Collectors.toList());
+            try (ThreadTrace trace = traceOnThread("tx.execute.get_group")) {
+                return executeInternal(query, infer);
+            }
         }
 
         public Stream<AnswerGroup<ConceptMap>> stream(GraqlGet.Group query) {
             return stream(query, true);
         }
 
         public Stream<AnswerGroup<ConceptMap>> stream(GraqlGet.Group query, boolean infer) {
-            Iterable<AnswerGroup<ConceptMap>> iterable = () -> rpcIterator(query, infer);
-            return StreamSupport.stream(iterable.spliterator(), false);
+            try (ThreadTrace trace = traceOnThread("tx.stream.get_group")) {
```

Comment:
Can we not use underscores or hyphens, and just use dots (`.`) to scope trace names? In this case, `tx.stream.get.group`, but this should be applied to all other trace names.